api_tests/resources/views/test_resource_detail.py

```diff
@@ -40,7 +72,8 @@ def test_status_code__unapproved__non_contributor(self, app, user_role):
             registration_state=RegStates.INITIAL, user_role=user_role
         )
         resp = app.get(make_api_url(test_artifact), auth=test_auth, expect_errors=True)
-        assert resp.status_code == 403 if test_auth else 401
+        expected_status_code = 403 if test_auth else 401
+        assert resp.status_code == expected_status_code
```

Comment:
Out of curiosity, why the change? Just better readability?

Reply:
Better error results on the tests. Having a condition on the RHS just leads to an `assert False` instead of showing the actual vs expected code. Need to go and fix this on the DELETEPermissions tests, too